test/e2e/apps/statefulset.go

```diff
@@ -762,7 +762,14 @@ var _ = SIGDescribe("StatefulSet", func() {
 			}, e2esset.StatefulPodTimeout, 2*time.Second).Should(gomega.BeNil())
 		})
 
-		ginkgo.It("should have a working scale subresource", func() {
+		/*
+						Release : v1.16
```

Comment:
Please reformat this comment.

Reply:
I've fixed the formatting, should be better now :smile: